src/VirtoCommerce.Platform.Core/ObjectValue/ObjectValueSerializedAsStringForIndexAttribute.cs

```diff
@@ -0,0 +1,6 @@
+using System;
+
+namespace VirtoCommerce.Platform.Core.ObjectValue
+{
+    public class ObjectValueSerializedAsStringForIndexAttribute : Attribute { }
```

Comment:
Maybe better to `params string[] PropertyNames` parameters to allow to specify properties that need to be converted using this attribute. In case of empty param names no properties converted.